modal/_utils/function_utils.py

```diff
@@ -190,6 +190,33 @@ def serialized_function(self) -> bytes:
             logger.debug(f"Serializing function for class service function {self.cls.__qualname__} as empty")
             return b""
 
+    def get_cls_vars(self) -> Dict[str, Any]:
+        if self.cls is not None:
+            cls_vars = {
+                attr: getattr(self.cls, attr)
+                for attr in dir(self.cls)
+                if not callable(getattr(self.cls, attr)) and not attr.startswith("__")
+            }
+            return cls_vars
+        return {}
+
+    def get_cls_var_attrs(self) -> Dict[str, Any]:
+        import dis
+
+        import opcode
```

Comment:
Are these heavy imports? We should already have loaded them anyway on client startup via `modal._serialization -> modal._vendor.cloudpickle`. Maybe we should care more about optimizing client startup time (although I think we need serialization utilities to do most things).

Reply:
They should be pretty light

Reply:
Cool, not reason not to import them at the top of the module then IMO